rllib/utils/numpy.py

```diff
@@ -247,3 +249,37 @@ def lstm(x,
             unrolled_outputs[:, t, :] = h_states
 
     return unrolled_outputs, (c_states, h_states)
+
+
```

Comment:
This convenience function replaces `to_float_array` everywhere and works for all tensor types (tf and torch and numpy).